src/dapp-tests/run-tests

```diff
@@ -3,3 +3,162 @@ export SOLC_FLAGS="--evm-version constantinople"
 for x in *.sol; do
   dapp test-standalone "$x"
 done
+
+error() {
+    printf "fail: function '%s' at line %d.\n" "${FUNCNAME[1]}"  "${BASH_LINENO[0]}"
```

Comment:
Maybe it would make sense if this went to stderr with `1>&2`?

Reply:
yeah, that's probably better